packages/mappings/src/tier-mapping.ts

```diff
@@ -1,7 +1,7 @@
 import { type Tier } from '@repo/database';
 
-export const maxUsersPerTier: Record<Tier, { maxUsers: number }> = {
-  Launch: { maxUsers: 1 },
+export const maxUsersPerTier: Record<Tier, { maxUsers: number; maxIntegrations?: number }> = {
+  Launch: { maxUsers: 1, maxIntegrations: 1 },
   Build: { maxUsers: 4 },
   Grow: { maxUsers: 10 },
   Scale: { maxUsers: 100 },
```

Comment:
```suggestion
  Build: { maxUsers: 4, maxIntegrations: Infinity },
  Grow: { maxUsers: 10, maxIntegrations: Infinity },
  Scale: { maxUsers: 100, maxIntegrations: Infinity },
```
This way we have the business on this file, and we don't need to do `const maxIntegrations = maxUsersPerTier[tierStatus].maxIntegrations ?? Infinity;`